content/tutorials/esc-setup-psp-creds/index.md

```diff
@@ -0,0 +1,128 @@
+
+Create a new folder in your shell (for example `esc-psp-tutorial`) and navigate into it. Then, run `pulumi new`, and navigate the menu to create a new stack from a template called `typescript`. Fill in the project name as `esc-psp-tutorial`, and the stack name as `%ORG NAME%/dev`, filling in your actual organization name (for the rest of the tutorial, `tutorials` will be used), and `dev` is the name of the new stack.
+
+Once Pulumi CLI finishes installing dependencies, you should see the following files in your tutorial folder:
+
+```
+Pulumi.yaml  index.ts  node_modules  package-lock.json  package.json  tsconfig.json
+```
+
+## Import the ESC Environment into the IaC stack
+
+In your tutorial folder, create a new file `Pulumi.dev.yaml`. This file will contain configuration specific to your new `dev` stack. Fill the file in with this:
+
+```
+environment:
+  - creds/psp
+```
+
+In the config files, `environment` field lets you specify an array of ESC Environments to import. In this case, we only want to import the Environment we created above, `creds/psp`, where `creds` is the ESC project name and `psp` is the Environment name.
+
+## Write the Pulumi program
+
+Open your new project in an IDE of your choice. Replace the code in `index.ts` with the code below:
+
+```typescript
+import * as service from "@pulumi/pulumiservice";
+import * as pulumi from "@pulumi/pulumi";
+
+let config = new pulumi.Config();
+
+var environment = new service.Environment("environment-from-psp", {
+  organization: "tutorials",
+  project: "esc-psp-tutorial",
+  name: "first-psp-environment",
+  yaml: new pulumi.asset.StringAsset(
+`values:
+  esc-mastered: true
+  psp-mastered: true`
+  )
+})
+```
+
+Remember to replace `tutorials` with your actual organization name.
+
+Now, to be able to use PSP, run `npm install @pulumi/pulumiservice` to install the plugin. (Your command will differ if you're not using `npm` as Node's package manager.)
+
+## Create a new Environment via IaC
+
+You are now all set! Run `pulumi up`, and your Pulumi program will create a new environment called `first-psp-environment`. Navigate to the Environments page in Pulumi Cloud console to confirm it exists. You should be able to see the environment definition from your code - both `esc-mastered` and `psp-mastered` values are now `true`.
+
+### Wrapping up
+
+You have learned how to create ESC environments in the Pulumi Cloud console, import them into IaC stacks and even create Environments using the Pulumi Service Provider. The environment you created, `creds/psp` can now be easily re-used between your stacks, or used to compose more complicated environments, using [Environment imports](https://www.pulumi.com/docs/esc/environments/imports/). Using what you know now, you can easily create other `creds` Environments for ther providers. For example, `creds/aws` or `creds/azure`!
```

Comment:
I would replace `complicated` with `complex` - has a more positive connotation

how about "Using what you know now, you can easily create other Environments under the `creds` project to store credentials for other services."